src/components/StaffingServices/StaffingServices.tsx

```diff
@@ -0,0 +1,95 @@
+import { Header } from "../Header/Header";
+import { Image } from "../Image/Image";
+import { Footer } from "../Footer/Footer";
+import { Button } from "../Button/Button";
+import outdoor from "../../assets/image-outdoor1.png";
+
+interface Props {
+  screenWidth?: "mobile" | "desktop";
+  onClick?: () => void;
+}
+
+export function StaffingServices({ onClick }: Props) {
+  return (
+    <>
+      <Header screenWidth="desktop" />
+      <div className="bg-wheat relative">
+        <h1 className="px-[16px] mt-[76px] pt-[60px] tracking-tighter text-welcome-title font-title text-center text-wetCoral leading-[86px] font-bold pb-[80px] md:text-mobile-title md:leading-[53px] md:pt-[80px] md:pb-[60px] md:px-[30px]">
+          Staffing Services
+        </h1>
+        <div className="flex justify-center">
+          <Image src={outdoor} alt="Staffing Services" context="staffing" />
+        </div>
+        <p className="tracking-tight pt-[80px] max-w-[782px] text-center mx-auto px-[16px] font-text text-paragraph-lg leading-[28px] font-normal text-lightBlack md:text-paragraph-sm md:leading-[26px] md:pt-[40px]">
+          Expert Event Solutions offers a variety of services to meet you needs.
+          Our team is experienced in planning, preparation, execution, customer
+          service, clean up and post event support for events ranging from
+          concerts, to art festivals, farmers markets, running and triathlon
+          events, weddings, corporate events and family functions.
+        </p>
+        <p className="tracking-tight pt-[40px] pb-[80px] max-w-[782px] text-center mx-auto px-[16px] font-text text-paragraph-lg leading-[28px] font-normal text-lightBlack md:text-paragraph-sm md:leading-[26px] md:pt-[20px] md:pb-[40px]">
+          Let us help you with your event. We can custom design a package to
+          provide you with as much support as you need from planning, and
+          permitting to staffing to ordering supplies or coordinating rentals.
+          We are here to help and love to see our supported events come off
+          without a hitch. Let us bring our experience to the table and help you
+          make the most of your event.
+        </p>
+        <div className="inline-block bg-gingerFlower rounded-[60px] flex flex-wrap justify-center relative mx-auto max-w-[1180px] pt-[80px] pb-[138px] px-[24px] xl:mx-[50px] lg:mx-[16px] md:py-[60px] md:px-0">
+          <div className="absolute bottom-[calc(80px-609px)] border-[6px] w-[1180px] px-[16px] h-[609px] border-gingerFlower rounded-b-[60px] xl:w-[calc(100vw-100px)] lg:w-[calc(100vw-32px)] md:h-[754px] md:top-[628px]"></div>
+          <ul className="flex flex-wrap w-fit px-[24px] justify-center lg:px-0 md:gap-[32px] md:flex-col">
```

Comment:
Is the reason you rolled your own `ul` here simply because we don't have the actual Horizontal list merged in yet?

Reply:
Yeah. Component was done today/yesterday and I finished the page 2 days ago.